### ISSUE TITLE
JsBindNet.js get fetchOption fail

### ISSUE THREAD

**Issue by yuzd** [mingyaulee/Blazor.BrowserExtension#28]
![image](https://user-images.githubusercontent.com/16083659/133892136-c5c04dc1-0ee9-4451-a311-95efaf9dab57.png)

```js
export async function Fetch(url, init) {
  const response = await fetch(url, init); //should change to 【  init.result 】
  return new FetchResponse(response);
}
```

![image](https://user-images.githubusercontent.com/16083659/133893066-5e76fe21-9bd2-46a3-b254-9e419324d007.png)

```js
  var fetchOptions = new Dictionary<string, object>(request.Options)
            {
                ["method"] = request.Method.Method,
// can not set fetchOption headers , so this line  should change to 【request.Content.Headers.ToDictionary(header => header.Key, header => string.Join(',', header.Value)) 】
                ["headers"] = request.Headers.ToDictionary(header => header.Key, header => string.Join(',', header.Value))

            };

```

**Comment by yuzd** [comment]
![image](https://user-images.githubusercontent.com/16083659/133892136-c5c04dc1-0ee9-4451-a311-95efaf9dab57.png)

**Comment by mingyaulee** [comment]
Great finding! I have pushed a fix in v0.6.4
